elevation/elevation-service.md

```diff
@@ -49,6 +49,8 @@ Without the `range`, the result looks something like this, with only a `height`:
 
 The `encoded_polyline` parameter is a string of a polyline-encoded, with six degrees of precision, shape and has the following parameters.
 
+You can find more information and [code samples for how to decode an encoded polyline](https://mapzen.com/documentation/mobility/decoding/) in the Mapzen Mobility documentation. 
```

Comment:
and then in the future we can add some samples of how to create an encoded polyline in various languages